cmds/boot/systemboot/main.go

```diff
@@ -18,6 +19,124 @@ import (
 	"github.com/u-root/u-root/pkg/smbios"
 )
 
+const (
+	_FB_OEM_IPMI_NET_FUNCTION = 0x36
+
+	_FB_OEM_SET_PROC_INFO = 0x10
+	_FB_OEM_SET_DIMM_INFO = 0x12
+
+	_WIWYNN_ID0     = 0x0
+	_WIWYNN_ID1     = 0x9c
+	_WIWYNN_ID2     = 0x9c
+	_MFGID_AENEON   = 0x5705
+	_MFGID_QIMONDA  = 0x5105
+	_MFGID_NEC      = 0x1000
+	_MFGID_IDT      = 0xB300
+	_MFGID_TI       = 0x9700
+	_MFGID_HYNIX    = 0xAD00
+	_MFGID_MICRON   = 0x2C00
+	_MFGID_INFINEON = 0xC100
+	_MFGID_SAMSUNG  = 0xCE00
+	_MFGID_TEK      = 0x3D00
+	_MFGID_KINGSTON = 0x9801
+	_MFGID_ELPIDA   = 0xFE02
+	_MFGID_SMART    = 0x9401
+	_MFGID_AGILENT  = 0xC802
+	_MFGID_NANYA    = 0x0B03
+	_MFGID_INPHI    = 0xB304
+	_MFGID_MONTAGE  = 0x3206
+	_MFGID_INTEL    = 0x8900
+	_MFGID_VIKING   = 0x4001
+)
+
+type memoryModuleManufacturerList struct {
```

Comment:
I would remove this struct and use a map[uint16]string for the ManufacturerID -> ManufacturerName mapping.

Reply:
Done